src/guides/v2.4/install-gde/prereq/mysql.md

```diff
@@ -180,6 +180,11 @@ The `explicit_defaults_for_timestamp` setting is deprecated. This setting contro
 {:.bs-callout-warning}
 On Magento projects deployed on the Cloud platform, the `explicit_defaults_for_timestamp` setting for MySQL (MariaDB) defaults to *OFF*
 
+{:.bs-callout-warning}
+Reindexing on MariaDB 10.4 takes more time compared to other MariaDB or MySQL versions. To speed up reindexing, we recommend setting these MariaDB configuration parameters:
```

Comment:
There should be a blank line after this line - this causes Linting errors if not present.

Reply:
No good way to put a list in a warning box via markdown.